### DISCUSSION TITLE
[Project Beta] Negated Draft Filter Shows Draft PRs

### DISCUSSION THREAD

**Discussion by helsaawy** [community/community#20530] [Projects and Issues]
The `-is:draft` filter shows draft PRs

![image](https://user-images.githubusercontent.com/84944216/179008976-d0611481-23b1-4c87-89a5-b1ca8a163746.png)

**Comment by LuigiImVector** [comment]
Try with `draft:false`

**Reply by helsaawy** [reply]
The project view filter does not recognize that as a valid query:

![image](https://user-images.githubusercontent.com/84944216/179052878-d69b1ab2-20ca-4f4e-8ca3-16e64e02b7a0.png)

**Reply by LuigiImVector** [reply]
You're right, add to this discussion the labels: Projects and Enhancement 

The NOT has not yet been implemented for the `is:`
